desktop-environment.el

```diff
@@ -357,22 +361,36 @@ portion of the screen."
 ;;; Commands - screenshots
 
 ;;;###autoload
-(defun desktop-environment-screenshot ()
+(defun desktop-environment-screenshot (delay)
   "Take a screenshot of the screen in the current working directory."
-  (interactive)
+  (interactive "P")
   (let ((default-directory (expand-file-name desktop-environment-screenshot-directory)))
-    (start-process-shell-command "desktop-environment-screenshot" nil desktop-environment-screenshot-command)))
+    (if (and delay
+             (numberp delay)
+             (> delay 0))
+        (start-process-shell-command "desktop-environment-screenshot"
```

Comment:
I would prefer just one call to `start-process-shell-command` instead of two, i.e.,

```emacs
(let ((default-directory ...)
       (command (if ...delay...)))
  (start-process-shell-command ...))
```
```

Reply:
Should be resolved as of  67b393f